frontend/src/lib-components/molecules/ThreadListItem.tsx

```diff
@@ -39,7 +39,8 @@ export const Container = styled.div<{ isRead: boolean; active: boolean }>`
   color: ${(p) => p.theme.colors.grayscale.g100};
   padding: ${defaultMargins.s} ${defaultMargins.m};
   cursor: pointer;
-  border: 1px solid ${(p) => p.theme.colors.grayscale.g15};
+  border-top: 1px solid ${(p) => p.theme.colors.grayscale.g15};
```

Comment:
The last message in the list now doesn't have a bottom border. Is that intentional?

Reply:
Yes. I think it looks better without a bottom border for the last item; the Figma design has so many messages the last message is not (fully) visible.